its/ruling/src/test/java/org/sonar/python/it/RulingHelper.java

```diff
@@ -143,6 +142,7 @@ static List<String> bugRuleKeys() {
       "S6323",
       "S6328",
       "S6468",
+      "S6559",
```

Comment:
I admit this is not properly documented (and it's not even very well designed at all) but this `RulingHelper` is meant to list only our bug rules, to be run on the extended ruling.

The rationale was that we thought our existing ruling was not sufficient to ensure proper confidence for bug rules, and yet we didn't want to bloat it and have too many projects on it.

We explicitly decided to only run bug rules on this extended ruling, so maybe it doesn't make sense to add this one here.

This whole approach can be challenged and we may want to do something about it, but for the time being I think we could stay consistent with this approach.

Reply:
done